### DISCUSSION TITLE
blog/deploy-and-config-website-on-name-cheap-part-1

### DISCUSSION THREAD

**Discussion by giscus** [hta218/leohuynh.dev#2] [Announcements]
# blog/deploy-and-config-website-on-name-cheap-part-1

Trước giờ mình luôn muốn sở hữu 1 site cá nhân, nhưng vì mình không có thời gian nên đợt này nghỉ việc, ở nhà cách ly mới tìm hiểu để làm được...

http://localhost:3000/blog/deploy-and-config-website-on-name-cheap-part-1

**Comment by hta218** [comment]
Looking good 👀
